rust-connectors/sources/syslog/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 [dependencies]
 syslog_loose = "0.17.0"
 structopt = "0.3"
-fluvio = "0.14"
+fluvio = "0.15"
```

Comment:
this can be changed to `0.1.0` since there was no release

Reply:
do you mean the version of syslog connector? If so, better to do it separately.